src/assets/styles/base/_base.scss

```diff
@@ -35,7 +35,84 @@ html {
   --color-paragraph: #{$color-coal};
   --color-share-icons: #{$color-majorelle};
   --color-social-icons: #{$color-cerulean};
+  --color-close-bg: #{$color-snow};
   --color-close: #{$color-cerulean};
+
+  @media screen and (min-width: map-get($breakpoints, 'breakpoint-11')) {
+    --color-close-bg: #{$color-ivory};
+  }
+}
+
+.page--holiday-extras {
+  --color-accent: #{$holiday-extras-gold};
+  --color-background: #{$holiday-extras-purple-3};
+  --color-heading-2: #{$color-smoke};
+  --color-heading-3: #{$holiday-extras-gold};
+  --color-intro-text: #{$color-smoke};
+  --color-text: #{$holiday-extras-purple-1};
+  --color-container: #{$holiday-extras-purple-4};
+  --color-focus-bg: #{$holiday-extras-gold};
+  --color-footer-contact: #{$color-smoke};
+  --color-footer-divider: #{$holiday-extras-purple-4};
+  --color-heading: #{$color-smoke};
+  --color-hero-subheading: #{$holiday-extras-purple-1};
+  --color-nav: #{$color-smoke};
+  --color-footer-text: #{$holiday-extras-purple-1};
+  --color-logo: #{$color-smoke};
+  --color-social-icons: #{$color-smoke};
+  --color-close-bg: #{$color-smoke};
+  --color-close: #{$holiday-extras-purple-5};
+  --color-metric-background: #{$holiday-extras-purple-2};
+  --color-metric-border: #{$holiday-extras-purple-4};
+  --color-metric-unit: #{$color-smoke};
+  --detail-heading: #{$holiday-extras-gold};
+  --color-detail-text: #{$color-white};
+  --color-detail-bg: #6F43B9;
```

Comment:
Why isn't this color in a variable? The hex format isn't in keeping with the Leaf coding standards (lowercase letters). Plus, in this project, we should be using HSL colors.